teuthology/salt.py

```diff
@@ -52,39 +53,45 @@ def __generate_minion_keys(self):
         for rem in self.remotes.iterkeys():
             minion_id = rem.hostname
             self.minions.append(minion_id)
-            log.debug("minion: ID {}".format(minion_id,))
+            log.debug('minion: ID {}'.format(minion_id,))
             # mode 777 is necessary to be able to generate keys reliably
-            # we hit this before: https://github.com/saltstack/salt/issues/31565
-            self.master_remote.run(args = [
+            # we hit this before:
+            # https://github.com/saltstack/salt/issues/31565
+            self.master_remote.run(args=[
                 'sudo',
                 'sh',
                 '-c',
-                'if [ ! -d salt ]; then\
-                mkdir -m 777 salt; fi'])
-            self.master_remote.run(args = [
+                ('if [ ! -d salt ]; then'
+                 'mkdir -m 777 salt; fi')
+            ])
+            self.master_remote.run(args=[
                 'sudo',
                 'sh',
                 '-c',
-                'if [ ! -d salt/minion-keys ]; then\
-                mkdir -m 777 salt/minion-keys; fi'])
-            self.master_remote.run(args = [
+                ('if [ ! -d salt/minion-keys ]; then'
+                 'mkdir -m 777 salt/minion-keys; fi')
+            ])
+            self.master_remote.run(args=[
                 'sudo',
                 'sh',
                 '-c',
-                'if [ ! -f salt/minion-keys/{mid}.pem ]; then\
-                salt-key --gen-keys={mid}\
-                --gen-keys-dir=salt/minion-keys/; fi'.format(mid = minion_id)])
+                ('if [ ! -f salt/minion-keys/{mid}.pem ]; then'
+                 'salt-key --gen-keys={mid}'
+                 '--gen-keys-dir=salt/minion-keys/;'
+                 ' fi').format(mid=minion_id)
+            ])
 
     def __cleanup_keys(self):
         '''
         Remove this cluster's minion keys (files and accepted keys)
         '''
         for rem in self.remotes.iterkeys():
             minion_id = rem.hostname
-            log.debug("Deleting minion key: ID {}".format(minion_id))
-            self.master_remote.run(args = ['sudo', 'salt-key', '-y', '-d',
-                '{}'.format(minion_id)])
-            self.master_remote.run(args = ['sudo', 'rm',
+            log.debug('Deleting minion key: ID {}'.format(minion_id))
+            self.master_remote.run(args=['sudo', 'salt-key', '-y', '-d',
+                                         '{}'.format(minion_id)])
```

Comment:
I don't agree with this one, since its not a 'hanging indent'. The closing ')' would end up way to far left to be associated easily with the arguments.